src/Module/Friendica.php

```diff
@@ -90,7 +94,8 @@ protected function content(array $request = []): string
 
 		$blockList = $this->config->get('system', 'blocklist') ?? [];
 
-		if (!empty($blockList)) {
+		$register_policy_int = intval($this->config->get('config', 'register_policy'));
```

Comment:
AFAIK the config functionality returns type safe values. So I guess there shouldn't be a use for `intval`, I guess.

Reply:
I copied the statement from the same file, I’ll change both places.